superset-frontend/src/SqlLab/components/SqlEditor/index.jsx

```diff
@@ -392,8 +397,7 @@ class SqlEditor extends React.PureComponent {
   canValidateQuery() {
     // Check whether or not we can validate the current query based on whether
     // or not the backend has a validator configured for it.
-    const validatorMap = window.featureFlags.SQL_VALIDATORS_BY_ENGINE;
```

Comment:
Same here - instead of checking the `featureFlags` map, we should check bootstrap data.